client/npm-shrinkwrap.json

```diff
@@ -825,272 +830,45 @@
       "from": "marked@>=0.3.5 <0.4.0",
       "resolved": "https://registry.npmjs.org/marked/-/marked-0.3.5.tgz"
     },
-    "node-libs-browser": {
```

Comment:
Had to remove node_modules, npm install and  npm shrinkwrap to install jQuery-ujs npm package and to get npm shrinkwrap to finish without errors.  

So I assuming it updated itself from that process referencing the package.json and installed the latest version?

> On Sep 10, 2015, at 11:42 PM, Justin Gordon <notifications@github.com> wrote:
> 
> In client/npm-shrinkwrap.json:
> 
> > @@ -825,272 +830,45 @@
> >        "from": "marked@>=0.3.5 <0.4.0",
> >        "resolved": "https://registry.npmjs.org/marked/-/marked-0.3.5.tgz"
> >      },
> > -    "node-libs-browser": {
> How did this all change from this PR?
> 
> —
> Reply to this email directly or view it on GitHub.
>